src/eterna/pose2D/RNALayout.ts

```diff
@@ -16,7 +21,7 @@ export class RNATreeNode {
     public goX: number = 0;
     public goY: number = 0;
 
-    public rotationDirectionSign: number = 1; // 1 or -1 for counterclockwise or clockwise
+    public rotationDirectionSign: RotationDirection;
```

Comment:
```suggestion
    public rotationDirection: RotationDirection;
```

Super minor thing!

Reply:
Oh, this will require modification wherever it's used, so you'll need to use VS Code's rename reference

Reply:
Sure, done.

Reply:
Done.